x-pack/plugins/observability/public/plugin.ts

```diff
@@ -46,6 +46,7 @@ import { createNavigationRegistry, NavigationEntry } from './services/navigation
 import { updateGlobalNavigation } from './update_global_navigation';
 import { getExploratoryViewEmbeddable } from './components/shared/exploratory_view/embeddable';
 import { createExploratoryViewUrl } from './components/shared/exploratory_view/configurations/utils';
+import { createUseRulesLink } from './hooks/create_use_rules_link';
```

Comment:
The increase in bundle size is probably coming from this `import`. We should move the import to an `await require` inside both the `setup` and `start` methods and declare those methods as `async`. Look at line 144 for the syntax.

Reply:
@simianhacker I did the change as suggested but CI still complains. Correct if I am wrong and blind, but in the Metrics table I see observability plugin 

87.5KB | 88.5KB | +1.0KB

which means that bundle size increased only by 1.0KB. Why is CI complaining then?

Reply:
@simianhacker I reverted the await import for createUseRulesLink. This didn't solve the metric bundle issue, plus I was getting a couple of typescript errors. 

This PRs includes one more new import `loadRules`, which sounds much more suspicious :-) I moved to an await import and now I am waiting for CI. Let's see !